relational_operators/WorkOrderFactory.cpp

```diff
@@ -420,22 +428,6 @@ WorkOrder* WorkOrderFactory::ReconstructFromProto(const serialization::WorkOrder
           shiftboss_client_id,
           bus);
     }
-    case serialization::WINDOW_AGGREGATION: {
```

Comment:
I forked this branch from the branch in #90, which in itself is quite old. Because of which some weird behavior may have happened. Thanks for pointing it out. I will remove any unrelated changes from this branch and let you know.